port/win/env_win.cc

```diff
@@ -789,7 +789,6 @@ class WinRandomAccessFile : public RandomAccessFile {
         random_access_max_buffer_size_(options.random_access_max_buffer_size),
         buffer_(),
         buffered_start_(0) {
-    assert(!options.use_mmap_reads);
```

Comment:
The use of WinRandomAccessFile is mutually exclusive with another class that is used for memory mapped file. That is why the assert is there so we do not instantiate the class that does not do memory mapped access when it's configured.